CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/MainActivity.java

```diff
@@ -59,7 +59,8 @@ public static String getLocalIpAddress() {
                 System.out.println(intf.getName());
                 for (Enumeration<InetAddress> enumIpAddr = intf.getInetAddresses(); enumIpAddr.hasMoreElements();) {
                     InetAddress inetAddress = enumIpAddr.nextElement();
-                    if (!inetAddress.isLoopbackAddress() && intf.getName().equals("eth1") && (inetAddress instanceof Inet4Address)) {
+                    String name = intf.getName();
+                    if (!inetAddress.isLoopbackAddress() && (intf.getName().equals("eth1") || intf.getName().equals("wlan0")) && (inetAddress instanceof Inet4Address)) {
```

Comment:
why don't you use the variable "name" defined to replace all intf.getName() statement?

Reply:
done.